trieste/models/model_interfaces.py

```diff
@@ -624,22 +624,37 @@ def randomize_hyperparameters(object: gpflow.Module) -> None:
             param.assign(param.prior.sample())
 
 
-def squeeze_hyperparameters(object: gpflow.Module, alpha: float = 1e-2) -> None:
+def squeeze_hyperparameters(
+    object: gpflow.Module, alpha: float = 1e-2, epsilon: float = 1e-7
```

Comment:
this seems like quite a high default for alpha?

Reply:
possibly, it's only 1% of the range though. Should I change to 1e-3? 1e-4? more?

Reply:
I meant that this could be a bit too big if used to translate (when using softplus without a lower bound)

Reply:
I use alpha for relative change (sigmoid) and epsilon for absolute change (softplus). It's extra complexity but I don't think we can find a single default value for both that would be satisfying.